pymc_marketing/mmm/transformers.py

```diff
@@ -260,6 +266,103 @@ def delayed_adstock(
     return batched_convolution(x, w, axis=axis)
 
 
+def weibull_adstock(
+    x,
+    lam=1,
+    k=1,
+    l_max: int = 12,
+    axis: int = 0,
+    type: WeibullType = WeibullType.PDF,
```

Comment:
I would use string or a boolean variable so users don't have to import an object to be able to parametrize the function

Reply:
+1

Reply:
We have a similar interface for [batched_convolution](https://github.com/pymc-labs/pymc-marketing/blob/main/pymc_marketing/mmm/transformers.py#L16)too. Maybe make it `type: WeibullType | str = WeibullType.PDF`?

Reply:
It will only every be pdf or not pdf (cdf). Think the boolean simplifies it heavily